### PR TITLE
chore(merge main): patched commit → 24cc552

### PR DESCRIPTION
**Pull request by julien-deramond**
- [x] https://github.com/twbs/bootstrap/commit/d867ee08b9fa87aaeab5f52c4b4b257485d5ecdb
- [x] https://github.com/twbs/bootstrap/commit/6922f5d9cb1e903877532f23214a4378aeaad909
- [x] https://github.com/twbs/bootstrap/commit/45fe28c5a6ed88d1a661355349aed799a79ed0fe
- [x] https://github.com/twbs/bootstrap/commit/f7f0026e4af2fee210b664e145dfe55bd12b7deb
- [x] https://github.com/twbs/bootstrap/commit/d1a648ab36f2679615e73617b838a48c19687966
- [x] https://github.com/twbs/bootstrap/commit/85d3c27816c393681c230a96996277bc5227aa75
- [x] https://github.com/twbs/bootstrap/commit/e1d67335436448bbd05b96648dae02d6dfb3b6a6
- [x] https://github.com/twbs/bootstrap/commit/3921236e6ec5e7ec28e57568d7954356d8988d97
- [x] ~https://github.com/twbs/bootstrap/commit/6a00d18fae8289e61cd3969c59b9215092a8dd84~: IDK what's used here exactly, so let our PNGs the way it is now. 
- [x] ~https://github.com/twbs/bootstrap/commit/859a1342033635f1cd8e790cfb9f698562022d0f~: IDK what's used here exactly, so let our SVGs the way it is now.
- [x] https://github.com/twbs/bootstrap/commit/1401722f7eed062e7c74a5a513c2cc0587043684
- [x] ~https://github.com/twbs/bootstrap/commit/a0912dc5434a2d27bd2af3c0da3b1a20c8f068e2~: N/A
- [x] https://github.com/twbs/bootstrap/commit/d1636a7540ae935ddd27462333c2bdf5a6eadef0
- [x] https://github.com/twbs/bootstrap/commit/11037f8407d2533bdb2d70962a1802d2ecfeadc9
- [x] https://github.com/twbs/bootstrap/commit/9480a3d01d12968f8c0b06cf62d9554120ae1639
- [x] https://github.com/twbs/bootstrap/commit/cd0484713c3239dbed88359d96b10a2c06d90891
- [x] https://github.com/twbs/bootstrap/commit/24cc552343c817b0d13201639c635a778d55da09

### CODE REVIEW

File: site/layouts/_default/examples.html [hunk 1]
```diff
@@ -89,74 +89,30 @@
       }
```
Comment (louismaximepiton) [inline site/layouts/_default/examples.html:89]: Missing a style change just above. Or maybe a `// Boosted mod: no ...` from https://github.com/twbs/bootstrap/commit/d1636a7540ae935ddd27462333c2bdf5a6eadef0

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
I think we may need some `defer: true` (when calling `extra_js`)

File: site/layouts/_default/examples.html (hunk 1 above)
Comment (julien-deramond) [reply site/layouts/_default/examples.html:89]: We have the style with the svg already. I can add a Boosted mod for this part yes

File: site/layouts/partials/stylesheet.html [hunk 2]
```diff
@@ -13,14 +13,19 @@
 
 {{- if (ne .Page.Layout "examples") }}
 {{- $targetDocsCssPath := path.Join "/docs" .Site.Params.docs_version "assets/css/docs.css" -}}
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/layouts/partials/stylesheet.html]: Good catch! Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2530/commits/14653a371992880a77d8dadb139119a4e4f936ab

**Review by louismaximepiton — APPROVED** [review]
👌

**Pull request merged**